connectors/activemq/build.gradle

```diff
@@ -0,0 +1,9 @@
+dependencies {
+    api project(':forklift-core'),
+            'org.apache.activemq:activemq-client:5.14.0',
+            'org.apache.activemq:activemq-broker:5.14.0',
```

Comment:
Do we want to push the broker code to everyone that uses the connector library?

Reply:
Good question. Let me look at that.